app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,34 @@
+class RecipesController < ApplicationController
+  def index
+    @recipes = current_user.recipes.where(user_id: params[:food_id])
```

Comment:
- [ ] Kindly display only recipes created by `current_user`. You may use @recipes = current_user.recipes